index.html

```diff
@@ -0,0 +1,304 @@
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+        </div>
+        <div class="social_media_about">
+          <a href="#">LET’S CONNECT</a>
+          <ul class="social_icons">
+            <li><a href="#"><i class="fa-brands fa-github fa-1x"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-linkedin-in fa-1x"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-angellist fa-1x"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-twitter fa-1x"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-m fa-1x"></i></a></li>
+          </ul>
+          <button type="submit" class="button-container">
+            Get my resume
+          </button>
+        </div>
+      </div>
+      <ul class="skill-list">
+        <li class="skills">
+          <h2>Languages</h2>
+          <div class="drop"><img src="images/drop-down.png" alt="drop-down-arrow" /></div>
+        </li>
+        <ul class="langList">
+          <li class="lang">
+            <picture>
+              <source media="(max-width: 768px)" srcset="images/js.png">
+              <source media="(min-width: 768px)" srcset="images/js2.png">
+              <img src="" alt="js-icon" />
+            </picture>
+          </li>
+          <li class="lang">
+            <picture>
+              <source media="(max-width: 768px)" srcset="images/html.png">
+              <source media="(min-width: 768px)" srcset="images/html2.png">
+              <img src="" alt="html-icon" />
+            </picture>
+          </li>
+          <li class="lang">
+            <picture>
+              <source media="(max-width: 768px)" srcset="images/css.png">
+              <source media="(min-width: 768px)" srcset="images/css2.png">
+              <img src="" alt="css-icon" />
+            </picture>
+          </li>
+        </ul>
+        <li class="skills">
+          <h2>Frameworks</h2>
+          <img src="images/drop-arrow.png" alt="drop-down-arrow" />
+        </li>
+        <hr />
+        <li class="skills">
+          <h2>Skills</h2>
+          <img src="images/drop-arrow.png" alt="drop-down-arrow" />
+        </li>
+        <hr />
+      </ul>
+    </div>
+
+    <!-- Contact Form  -->
+    <section id="Contact" class="contact-me">
+      <h2>Contact me</h2>
+      <p>
+        If you have an application you are interested in developing, a feature that you need
+        built or a project that needs coding. I’d love to help with it
+      </p>
+      <form action="https://formspree.io/f/mvoyvrlg" method="POST" id="contactme">
+        <input type="text" id="name" name='name' maxlength="30" placeholder="Enter your name" required />
+        <input type="email" id="email" name="email" placeholder="Enter your email" required />
+        <small></small>
+        <textarea id='message' name="message" maxlength="500" placeholder="Type your message here" required></textarea>
```

Comment:
- [ ] Please increase the padding of this element so as to ensure great user interaction. 😄